README.rst

```diff
@@ -83,7 +83,7 @@ FAQ
 How to create a top level .Trash dir?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Steps ::
+Steps::
```

Comment:
The correction of "rhe" in "the" is fine, but please send me a pull request without this other change of this space.